ArduCopter/arming_checks.cpp

```diff
@@ -75,6 +75,21 @@ bool Copter::pre_arm_checks(bool display_failure)
         return true;
     }
 
+    bool ret = true;
+    ret &= rc_calibration_checks(display_failure);
+    ret &= compass_checks(display_failure);
+    ret &= gps_checks(display_failure);
+    ret &= fence_checks(display_failure);
+    ret &= ins_checks(display_failure);
+    ret &= board_voltage_checks(display_failure);
+    ret &= parameter_checks(display_failure);
+    ret &= pilot_throttle_checks(display_failure);
+
+    return ret;
+}
+
+bool Copter::rc_calibration_checks(bool display_failure)
```

Comment:
This says rc_calibration_checks but it also looks for baro. I think this should be separated into two functions.